### PR TITLE
Updated Spotify to version 1.1.72.439.gc253025e.

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

------

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34667)

### CODE REVIEW

File: manifests/s/Spotify/Spotify/1.1.72.439.gc253025e/Spotify.Spotify.installer.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 PackageIdentifier: Spotify.Spotify
-PackageVersion: 1.1.71.560.gc21c3367
+PackageVersion: 1.1.72.439.gc253025e
```
Comment (OfficialEsco) [inline manifests/s/Spotify/Spotify/1.1.72.439.gc253025e/Spotify.Spotify.installer.yaml:3]: I'm getting version 1.1.71.560.gc21c3367, also the hash didn't change so guess its correct

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/s/Spotify/Spotify/1.1.72.439.gc253025e/Spotify.Spotify.installer.yaml (hunk 1 above)
Comment (jedieaston) [reply manifests/s/Spotify/Spotify/1.1.72.439.gc253025e/Spotify.Spotify.installer.yaml:3]: Yeah, kind of figured it downloaded the latest version on start. Oh well.

Comment (jedieaston) [reply manifests/s/Spotify/Spotify/1.1.72.439.gc253025e/Spotify.Spotify.installer.yaml:3]: Looks like it took them a minute.

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by jedieaston** [comment]
cc @jamespik  (sorry for the bump, but this hash has been bad for three days)

**Pull request merged**